examples/helmAgentAddon/example/policyaddon/chart/templates/policy.open-cluster-management.io_configurationpolicies_crd.yaml

```diff
@@ -0,0 +1,387 @@
+{{- if semverCompare "< 1.16.0" .Capabilities.KubeVersion.Version }}
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
i think CRDs are expected to be placed in a dedicated folder named `crds`, is this supported?

Reply:
I missed this feature. have supported it . thanks